.changeset/honest-jars-report.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
```

Comment:
it's interesting that there are three changesets for this PR. I've never seen a PR with multiple, so don't quite know what will happen when we squash this

Reply:
three changelog entries